data/translation/Zed/ConfigurableBundles/de_DE.csv

```diff
@@ -0,0 +1,2 @@
+configurable_bundle.template.name.1,"Konfigurierbares Bündle ""All in"""
```

Comment:
Typo in Bündle (for German)
needs fixing as follow up on nonsplit later

Reply:
I've reported it, thx